runtime/libffi/z/ffi64.c

```diff
@@ -811,7 +811,6 @@ ffi_check_struct_for_complex(ffi_type *arg_type)
   }
   return arg_type->type;
 }
-
```

Comment:
Please don't delete separating blank lines such as this. There should also be an "end" marker like other functions:
```c

/*======================== End of Routine ============================*/

```